api/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import app from "./app"
+
+const PORT: string = process.env.PORT || "4000"
```

Comment:
I wonder if we need to pick a different port, the e2e test docker container runs on port 4000